### PR TITLE
Attempting to follow PEP8 coding style.

### PR DESCRIPTION
**Pull request by Tri125**
I did this quickly (didn't have Gwent beside me to test run it). I didn't attempt to change any behavior so if something changed/broke, that's 100% a mistake.

I only followed what my IDE (pycharm) was suggesting me to do concerning code style.

### CODE REVIEW

**Comment by Tri125** [comment]
Btw this is a PR for the mechanics branch, not master.

File: CardData.py [hunk 1]
```diff
@@ -77,9 +77,12 @@
...
 
+
 class CardData:
-    def __init__(self, gwentDataHelper):
-        self._helper = gwentDataHelper
+    def __init__(self, gwent_data_helper):
+        helper = gwent_data_helper
+        self.patch = None
+        self.imageUrl = None
         self.cardTemplates = self._helper.getCardTemplates()
```
Comment (Zolmeister) [inline CardData.py]: broken, also avoid renaming the var to `helper`

File: CardData.py [hunk 2]
```diff
@@ -77,9 +77,12 @@
...
         self.tooltipData = self._helper.getTooltipData()
```
Comment (Zolmeister) [inline CardData.py]: instance vars and `gwent_data_helper` methods still camel cased

File: GwentUtils.py [hunk 3]
```diff
@@ -7,76 +7,79 @@
...
         path = self._folder + "tooltips_" + locale + ".csv"
         if not os.path.isfile(path):
             print("Couldn't find " + locale + " tooltips at " + path)
             exit()
         return path
 
-    def getCardTooltips(self, locale):
-        tooltipsFile = open(self.getTooltipsFile(locale), "r")
+    def get_card_tooltips(self, locale):
+        tooltips_file = open(self.getTooltipsFile(locale), "r")
```
Comment (Zolmeister) [inline GwentUtils.py]: again, breaks camel_case

**Review by Zolmeister — CHANGES_REQUESTED** [review]

File: CardData.py (hunk 1 above)
Comment (Tri125) [reply CardData.py]: ``self._helper`` was an object variable that is only used inside the constructor. That's why I made it a regular local variable.

**Comment by Tri125** [comment]
Good catch, it seems that my IDE isn't able to process ``self`` properly. I'll do a second pass. After that you guys should feel free to directly edit the PR as I will be unable to apply any new requested changes in a timely matter for the next 20h.

File: CardData.py (hunk 1 above)
Comment (Zolmeister) [reply CardData.py]: right, delete it and use `gwent_data_helper` directly

File: CardData.py [hunk 4]
```diff
@@ -181,62 +184,66 @@ def _createBaseCardJson(self):
 
                 art = {}
                 for imageSize in IMAGE_SIZES:
```
Comment (Zolmeister) [inline CardData.py]: loop variables still camel-cased

Comment (Tri125) [reply CardData.py]: AH, I thought this was accepted. Alright, give me a sec.

**Comment by Tri125** [comment]
Alright, I'm signing off for tonight. Edit the PR directly if you want changes before the end of the day.

**Review by Zolmeister — APPROVED** [review]

**Pull request merged**